source/Playnite/Localization/LocSource.xaml

```diff
@@ -762,21 +762,48 @@ Version {2}
     
 {0}</sys:String>
 
+    <!--Battle.net-->
+    <sys:String x:Key="LOCBattleNetLinksPCGW">PCGamingWiki</sys:String>
+
+    <!--Bethesda-->
+    <sys:String x:Key="LOCBethesdaLinksPCGW">PCGamingWiki</sys:String>
+
     <!--itch.io-->
     <sys:String x:Key="LOCItchioSignInNotif">itch.io client will now open. Please sign in and then close this message.</sys:String>
     <sys:String x:Key="LOCItchioSignInWaitMessage">Waiting for user to sign in, please close this when you're done…</sys:String>
     <sys:String x:Key="LOCItchioClientNotInstalledError">Itch client version 25 or newer is not installed.</sys:String>
+    <sys:String x:Key="LOCItchioLinksPCGW">PCGamingWiki</sys:String>

```

Comment:
We don't need localization string for PCGamingWiki, it's a website name, nobody is going to translate it. And we definitely don't need duplicated version of that string for every library.

Reply:
reverted to static string